pkgs/applications/video/mpv/scripts/chapterskip.nix

```
@@ -14,7 +14,13 @@ buildLua {
   };
 
   meta = {
+    description = "Automatically skips chapters based on title";
+    longDescription = ''
+    MPV script that skips chapters based on their title, categorized using regexes.
+    The set of skipped categories can be configured globally, or by an auto-profile.
+    '';
     homepage = "https://github.com/po5/chapterskip";
+    license = lib.licenses.unfree;  # po5/chapterskip#10
```

```suggestion
    license = lib.licenses.unfree;  # https://github.com/po5/chapterskip/issues/10
```

let's make this a proper link

---

Nice! I might want to check if it's possible to configure my editor to do that as well. It's not clickable in code in GitHub UI though (only in comments and issue/PR bodies), so I'd personally prefer something that can be copied easily in all circumstances.